lib/azure/resource_management/vnet_config.rb

```diff
@@ -0,0 +1,254 @@
+#
+# Author:: Aliasgar Batterywala (aliasgar.batterywala@clogeny.com)
+#
+# Copyright:: Copyright (c) 2016 Opscode, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'ipaddress'
+
+module Azure::ARM
+  module VnetConfig
+
+    ## lists subnets of only a specific virtual network address space ##
+    def subnets_list_specific_address_space(address_prefix, subnets_list)
```

Comment:
The method name can more readable so that we don't need to add comments like this method lists subnets of only a specifc vnet address space. So method names can be specific_vnet_address_subnet_list or something similar.